primitives/primitives/src/types.rs

```diff
@@ -56,4 +56,7 @@ pub type Signature = MultiSignature;
 /// Value type for price feeds.
 pub type Value = u128;
 
+/// Identifier for a SAFT
+pub type SAFTId = u32;
```

Comment:
New type missed in `types.json`

Reply:
Good catch,
Can we add this to the checklist aswell, I've got a feeling that this will happen a lot more

Reply:
The `e2e` tests check this automatically =.=